README.md

```diff
@@ -126,9 +182,14 @@ curl -X PUT https://api.us-east-1.mbedcloud.com/v2/notification/callback \
 }'
 ```
 
-Where `PELION_API_KEY` is the API key from Pelion Device Management portal, `AWS_API_ENDPOINT` and `AWS_API_KEY` are the API endpoint and key retrieved from AWS API Gateway above. 
+Where `PELION_API_KEY` is the access key from Pelion Device Management portal, `AWS_API_ENDPOINT` and `AWS_API_KEY` are the API endpoint and key retrieved from AWS API Gateway earlier. 
+
+If all goes well, you should get a **204** response back.
+
+- Next, enable pre-subscriptions for LwM2M resources that you want to be stored in Elasticsearch:
```

Comment:
Subscripts and presubscription can be managed with the ['v2/subscriptions'](https://developer.pelion.com/docs/device-management-api/connect/#/Device%20data%20-%20subscriptions) REST API endpoint on Pelion.

Reply:
added in 4b755d7